docker_ssh.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
This seems like it could just be reproduced where it's needed, it's pretty short.